src/mlpack/methods/preprocess/preprocess_split_main.cpp

```diff
@@ -67,7 +68,19 @@ BINDING_EXAMPLE(
     "\n\n" +
     PRINT_CALL("preprocess_split", "input", "X", "input_labels", "y",
         "test_ratio", 0.3, "training", "X_train", "training_labels", "y_train",
-        "test", "X_test", "test_labels", "y_test"));
+        "test", "X_test", "test_labels", "y_test") +
+    "Also by default the dataset is not stratified; you can provide the " +
```

Comment:
If this example is not related with above example, then for improving the readability, we can split the example into two different macros(`BINDING_EXAMPLE()`).
For E.g.
```cpp
BINDING_EXAMPLE(
    "So, a simple...");
BINDING_EXAMPLE(
    /*Stratify_data Example*/);
```
Let me what do you think?

Reply:
You are right @Yashwants19. The examples are not related and it makes sense to split them up. Will make this change.